library/src/scripts/navigation/siteNavAdminLinksStyles.ts

```diff
@@ -46,5 +46,11 @@ export const siteNavAdminLinksClasses = useThemeCache(() => {
         ...debug.name("link"),
     });
 
-    return { root, item, divider, link };
+    const linkIcon = style({
+        marginLeft: px(-6),
```

Comment:
Remove negative margin and get rid of this one instead: https://github.com/vanilla/vanilla/blob/master/library/src/scripts/navigation/siteNavAdminLinksStyles.ts#L45